Libraries/LibWeb/CSS/CSSRuleList.cpp

```diff
@@ -130,7 +129,8 @@ WebIDL::ExceptionOr<unsigned> CSSRuleList::insert_a_css_rule(Variant<StringView,
         break;
     }
 
-    if (rule_is_disallowed)
+    // FIXME: There are more constraints that we should check here - Parser::is_valid_in_the_current_context is probably a good reference for that.
```

Comment:
I think ideally we'd have a single place that handles the constraints, so that we don't have multiple places to remember to change when a new rule is added, but I'm not sure that would actually work in practice. If you come up with something that would be great though!